src/test/scala/chiselTests/BundleSpec.scala

```diff
@@ -172,6 +173,27 @@ class BundleSpec extends ChiselFlatSpec with BundleSpecUtils with Utils {
     }
   }
 
+  "Calling litValue on Bundle containing DontCare" should "make a decent error message" in {
+    class MyBundle extends Bundle {
+      val a = UInt(8.W)
+      val b = Bool()
+      val c = UInt(4.W)
+    }
+
+    class Example extends RawModule {
+      val out = IO(Output(new UInt()))
```

Comment:
```suggestion
      val out = IO(Output(UInt()))
```
Coding via Github reviews is such a drag.